etc/youtube-dl.profile

```diff
@@ -45,8 +52,11 @@ seccomp
 shell none
 tracelog
 
+disable-mnt
```

Comment:
media on other drives is common enough use case?
most other media profiles don't have this

Reply:
ytdl is only for downloading.

Reply:
@SkewedZeppelin Nice catch, fixed.

Reply:
I still don't see a reason, why ytdl should can access other drivers.

Reply:
@rusty-snake 
> I still don't see a reason, why ytdl should can access other drivers.

Suppose a user wants to download to a USB-drive mounted at /mnt/my-youtube-usb-stick/foo that will fail with `disable-mnt`. Another reason why @SkewedZeppelin is correct would be that youtube-dl supports `--external-downloader foo`, which also should be able to save straight to a device mounted under /mnt. Or do you have something else in mind?